tests/clients/layer0_test_client.go

```diff
@@ -2,11 +2,14 @@ package clients
 
 import (
 	"testing"
+	"time"
 
-	"github.com/quintilesims/layer0/cli/client"
+	"github.com/quintilesims/layer0/client"
 	"github.com/quintilesims/layer0/common/models"
 )
 
+var defaultTimeout = time.Minute * 5
```

Comment:
This already exists in the `config` package. Also, fwiw, 5 minutes isn't long enough.